DTS-STN/Scrum-Poker
# Added client side button functionality

## [DUMS-56 - Client side functionality when buttons are pressed](https://jira-dev.bdm-dev.dts-stn.com/browse/DUMS-56) (Jira Issue)

### Description

This PR adds client side functionality for the buttons on the room page.

### What to test for/How to test

1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/room/1`
4. Select a card from the list so the change is reflected in the user list
5. Click the hide button to change the card in the user list to the card back
6. Click show button to change the card in the user list back to the selected card
7. Click clear button to clear the selected card, which should change to the empty card SVG in the user list
8. After clicking the clear button, click hide button again to ensure that it does not make the card back appear

### Additional Notes

Perhaps look into adding an animation when switching between show and hide?
